scot/tests/test_ooapi.py

```diff
@@ -238,6 +238,25 @@ def test_plotting(self):
                 api.get_tf_connectivity('PHI', winlen=2, winstep=1, plot=fig)
                 api.compare_conditions([0], [1], 'PHI', plot=fig, repeats=5)
 
+    def test_random_state(self):
+        np.random.seed(10)
+        api = scot.Workspace(VAR(1),locations=[[0, 0, 1], [1, 0, 0], [0, 1, 0], [-1, 0, 0], [0, -1, 0]], reducedim=None)
+        api.set_data(np.random.randn(10, 5, 10), [1, 0] * 5)
+
+        # test MVARICA
+        api.do_mvarica(random_state=1)
+        mixing1 = api.mixing_
+        api.do_mvarica(random_state=1)
+        mixing2 = api.mixing_
+        assert_allclose(mixing1, mixing2)
+
+        # test CSPVARICA
+        api.do_cspvarica(random_state=1)
+        mixing1 = api.mixing_
+        api.do_cspvarica(random_state=1)
+        mixing2 = api.mixing_
+        assert_allclose(mixing1, mixing2)
```

Comment:
Why do you use `assert_allclose` instead of `assert_allequal`? Since we put the same numbers in the algorithms, there should be no numerical differences...